src/vs/workbench/browser/parts/editor/editorActions.ts

```diff
@@ -1925,10 +1927,12 @@ export class ReopenResourcesAction extends Action {
 	}
 }
 
-export class ToggleEditorTypeAction extends Action {
+export class ShowTextEditorAction extends Action {
+
+	static readonly ID = 'workbench.action.reopenTextEditor';
+	static readonly LABEL = localize('workbench.action.reopenTextEditor', "Reopen Editor With Text Editor");
+	private readonly fileEditorInputFactory = Registry.as<IEditorInputFactoryRegistry>(EditorExtensions.EditorInputFactories).getFileEditorInputFactory();
 
-	static readonly ID = 'workbench.action.toggleEditorType';
-	static readonly LABEL = localize('workbench.action.toggleEditorType', "Toggle Editor Type");
 
```

Comment:
We can add this new command but toggle is also useful, specifically for the markdown preview case

Reply:
Please report as separate issue to follow up with.